content/sensu-go/5.16/guides/email-handler.md

```diff
@@ -0,0 +1,204 @@
+---
+title: "Send email alerts with the Sensu Go Email Handler"
+linkTitle: "Send Email Alerts"
+description: "Here’s how to send alerts to your email with the Sensu Go Email Handler. Use handlers to send events to your technology of choice (in this case, email) to alert you of incidents and help you resolve them more quickly."
+weight: 85
+version: "5.16"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.16:
+    parent: guides
+---
+
+- [Create a check](#create-a-check)
+- [Add the email handler asset](#add-the-email-handler-asset)
+- [Add an event filter](#add-an-event-filter)
+- [Assign the email handler to a check](#assign-the-email-handler-to-a-check)
+- [Create an email template](#create-an-email-template)
+- [Next steps](#next-steps)
+
+Sensu event handlers are actions the Sensu backend executes on [events][1].
+This guide explains how to use the [Sensu Go Email Handler][3] to send incident notification emails.
+
+To use this guide, you’ll need to [install the Sensu backend][12] and have at least one [Sensu agent][13] running on Linux.
+You should also [install and configure sensuctl][4].
+
+## Create a check
```

Comment:
Theoretically, we don't even need to create a check--we could just create an ad-hoc event through the agent API, which might be a bit quicker than the CPU check, as folks will need to wait for CPU usage to go above the threshold or install a tool to generate load. Generating an event through the API would allow folks to receive an email quicker.

Reply:
Yes, I wasn't sure which path to take with this. I realized that the check-cpu check wouldn't do anything right away, so it wouldn't allow readers to get an email right away (maybe never if cpu state never changes, heh).

But going through all the steps to prepare a manually triggered event seemed like a distraction from the purpose of the guide (setting up the email handler). I think a better idea for this guide would be a check that *does* work right away -- check-cpu is just convenient because we already told them how to create it in a separate guide.

In this case, we have the Up and Running Katacoda scenarios, which does include a manually triggered event that works as a live demo. I could make that clearer here...something like "if you don't want to wait for a CPU state change, try the scenario..."

Reply:
I'm partial to the ad-hoc event approach because we avoid hand-waving about how to create a failing event, helping users stay focused on the email alert. We also get to remove the requirement for a running Sensu agent. 

I think using `eval $(sensuctl env)` and a curl command or two for the ad-hoc events should feel pretty straight-forward for our audience.

Instead of reference our other guide as a required step, we could move it to the conclusion, e.g. "In this guide you'll learn how to trigger an email from Sensu using ad-hoc events.", and "Now that you've completed this, the same email handler can be reused in check configurations like `check-cpu` from our `Monitor server resources` guide."

Reply:
Thank you both -- the updated guide uses the ad hoc event.